ui/less/containers.less

```diff
@@ -256,6 +256,13 @@
     color: white;
     cursor: pointer;
   }
+
+  .shaka-current-selection-span {
+    align-items: flex-start;
+
+    color: white;
+    cursor: pointer;
```

Comment:
I think this attributes are already inherited to `this.stateSpan_` (in `ui/statistics_button.js`) via their parent label in `.shaka-context-menu label`

https://github.com/google/shaka-player/blob/db8ad31bfbe1c2b9a372daad893d900e77d5e484/ui/less/containers.less#L251-L258

Reply:
That is not the case, because of how it is built, `shaka-current-selection-span` has higher priority, plus for menu overflow it is necessary. Therefore, to maximize compatibility between the two, you need to add this.